components/ResultsPageComponents/TopThree/TopThree.jsx

```diff
@@ -26,7 +26,11 @@ export default function TopThree({
     >
       <Flex direction="column">
         <Text fontWeight={600} fontSize="33px" lineHeight="37px" py="15px">
-          Top Three
+          Top Three travel methods
```

Comment:
let's capitalise the words in the headings, to stay consistent with other section headings
```suggestion
          Top Three Travel Methods
```